### ISSUE TITLE
Make Crosswalk Optional

### ISSUE THREAD

**Issue by kamilersz** [rstoenescu/quasar-framework#8]
Can I remove crosswalk from cordova?

I know that removing crosswalk may cause apps not working on older device. But there might be other consideration such as making smaller footprint by removing crosswalk.

**Comment by rstoenescu** [comment]
I will add an option to the command, but until then you can remove Crosswalk:

``` bash
$ quasar wrap plugin remove cordova-plugin-crosswalk-webview
```

**Comment by rstoenescu** [comment]
Released Quasar CLI v0.1.3 which adds this option to skip Crosswalk. Read release notes [here](http://quasar-framework.org/guide/quasar-cli-release-notes.html).
Thanks for the suggestion!
